check-version-bump.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+
+VERSION_CHANGE_TRIGGERS="setup.py MANIFEST.in openfisca_france"
+
+current_version=`python setup.py --version`
+
+if git diff-index --quiet origin/master $VERSION_CHANGE_TRIGGERS ":(exclude)*.md"
+then exit 0  # there are no functional changes at all, the version is correct
+fi
+
+if git rev-parse --verify --quiet $current_version
+then
+    echo "Version $current_version already exists:"
+    git log -1 $current_version
```

Comment:
But I don't think that should be blocking in a non-interactive shell. Is it blocking when you execute the script, or when you run `git log` directly in your console?